src/act.core.web/Views/Nodes/NodeSearchResultPartial.cshtml

```diff
@@ -16,7 +16,7 @@
                     <li class="text-right text-muted">
                         <div class="text-blue">Last Compliance Result Date</div>
                     </li>
-                    <li class="text-center text-muted"><small>@(Model.LastComplianceDate.HasValue? Model.LastComplianceDate.Value.ToLocalTime().ToString("MM-dd-yyyy hh:mm tt"): "Not Available")</small></li>
+                    <li class="text-center text-muted"><small>@(Model.LastComplianceDate.HasValue ? Convert.ToDateTime(Model.LastComplianceDate.Value).AddMinutes(-1 * ViewBag.TimeOffset).ToString("MM-dd-yyyy hh:mm tt") : "Not Available")</small></li>
```

Comment:
I am assuming LastComplianceDate is stored in UTC?  Also would it be better to create a readonly property in the Model class (something like LastComplianceDateLocal) instead of putting this in the view?

Reply:
Yes Tom. LastComplianceDate is in UTC.